packages/core/addon/navi-visualization-manifests/goal-gauge.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Visualization Manifest File for the Goal Gauge Visualization
+ * This file registers the visualization with navi
+ */
+import NaviVisualizationBaseManifest from './base';
+import RequestFragment from 'navi-core/models/bard-request-v2/request';
+
+export default class GoalGaugeManifest extends NaviVisualizationBaseManifest {
+  name = 'goal-gauge';
+
+  niceName = 'Goal Gauge';
+
+  icon = 'tachometer';
+
+  typeIsValid(request: RequestFragment) {
+    return this.hasNoGroupBy(request) && this.hasSingleTimeBucket(request) && this.hasSingleMetric(request);
```

Comment:
What happens if you have no time filters or columns?

Reply:
There's kind of a lot of references to jump through but this case should return false since it requires a single time bucket which requires an `interval` and `timeGrain`
https://github.com/yahoo/navi/blob/431e2714462f222a9d9eff61a07d4404b9d848ae/packages/core/addon/navi-visualization-manifests/base.ts#L51-L55
https://github.com/yahoo/navi/blob/431e2714462f222a9d9eff61a07d4404b9d848ae/packages/core/addon/navi-visualization-manifests/base.ts#L42-L46

`interval` and `timeGrain` are only defined in the case of having both a timeDimension filter and column
https://github.com/yahoo/navi/blob/431e2714462f222a9d9eff61a07d4404b9d848ae/packages/core/addon/models/bard-request-v2/request.ts#L193-L196
https://github.com/yahoo/navi/blob/431e2714462f222a9d9eff61a07d4404b9d848ae/packages/core/addon/models/bard-request-v2/request.ts#L209-L213

Although this can probably support more cases since it's trying to drill down to a single row of data with 1 metric